__tests__/app.test.js

```diff
@@ -74,3 +74,31 @@ describe("GET/api/topics",()=>{
             })
         })
     })
+
+describe("5. GET /api/articles/:article_id",()=>{
+    test('should return 200 with the article with the specified id',()=>{
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(data=>{
+            expect(data.body.article[0]).toEqual(
+                {
```

Comment:
We need to make sure that the expected keys have the exact correct values here - must be values related to the requested review_id, e.g. NOT review_id: expect.any(Number)